internal/server/statvar_search.go

```diff
@@ -53,6 +55,12 @@ func (s *Server) SearchStatVar(
 		svList = filter(svList, statExistence, len(places))
 		svgList = filter(svgList, statExistence, len(places))
 	}
+	if len(svList) > maxNumResult {
+		svList = svList[0:maxNumResult]
+	}
+	if len(svgList) > maxNumResult {
+		svgList = svgList[0:maxNumResult]
+	}
```

Comment:
as a todo, return the total number of results available? so we can add text to the effect of "x more stat vars"

Reply:
Done